Add --keep option to allow to generate newsfile, but keep newsfragmen… (#453)
twisted/towncrier: docs/cli.rst

   Do not ask for confirmations.
   Useful for automated tasks.

.. option:: --keep

>    Do not ask for confirmations. But keep news fragments.
>    Useful for automated tasks. Where you have to generate several different news fragments for different parties.
> 

Reviewer: I think the important part here is that fragments are not removed.

I don't know if "Useful for automated tasks" adds any new info on top of "Don't ask for confirmation".
If it doesn't ask, it should be good for automation.

I don't know what to day about the specific use case.
From this one sentence, I don't fully undestand what "Where you have to generate several different news fragments for different parties." should do.

If we want to have this information in the docs,
I think we should create a separate doc page with use case scenarios.

Something similar to "How to keep changelong in markdown"

In this way we can add more info.

I guess that you are using towncrier to generate a "Release-Notes-Everyting" and a separate "Release-Notes-Bugs" and "Release-Notes-Security" and "Release-Notes-Features"... but it is not clear.

```suggestion
   Don't delete news fragments after the build and don't ask for confirmation whether to delete or keep the fragments.

```
